kubernetes-extension-test/src/test/java/org/ballerinax/kubernetes/test/IstioVirtualServiceTest.java

```diff
@@ -0,0 +1,668 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerinax.kubernetes.test;
+
+import io.fabric8.docker.api.model.ImageInspect;
+import org.ballerinax.kubernetes.exceptions.KubernetesPluginException;
+import org.ballerinax.kubernetes.test.utils.KubernetesTestUtils;
+import org.ballerinax.kubernetes.utils.KubernetesUtils;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.wso2.andes.util.FileUtils;
+import org.yaml.snakeyaml.Yaml;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Map;
+
+import static org.ballerinax.kubernetes.KubernetesConstants.DOCKER;
+import static org.ballerinax.kubernetes.KubernetesConstants.KUBERNETES;
+import static org.ballerinax.kubernetes.test.utils.KubernetesTestUtils.getDockerImage;
+
+/**
+ * Test cases for generating istio virtual service artifacts.
+ *
+ * @since 0.983.0
```

Comment:
The version should be changed to '0.985'

Reply:
Fixed with https://github.com/ballerinax/kubernetes/pull/220/commits/5f8887c1388816ebe18d4066c627a7d43a88bea9